docs/configuration.md

```diff
@@ -394,7 +392,7 @@ And, conversely, the following would only format Jupyter Notebook files and not
     [tool.ruff]
     extend-include = ["*.ipynb"]
 
-    [tool.ruff.lint]
```

Comment:
Using `exclude` under `lint` is correct here. We only exclude `.ipynb` files from linting, but keep them as part of formatting (same as the example aboth).

Reply:
`include` fails even with the latest version, so the documentation is wrong and needs fixing.

`exclude` fails only if you happen to have ruff 0.1.0 (or earlier) installed, but I do not know what the backwards compatibility story is here. If it is okay to assume users have a very recent ruff installed, then using it here is fine.

My stake in this is as someone who doesn't use the formatter at all, but found this page as the first and most obvious place discussing how to configure the linter.

Reply:
Sorry, I should rephrase that.

> `include` fails even with the latest version, so the documentation ***in general*** is wrong and needs fixing.
> 
> `exclude` fails only if you happen to have ruff 0.1.0 (or earlier) installed, but I do not know what the backwards compatibility story is here. If it is okay to assume users have a very recent ruff installed, then using it here ***on this specific line*** is fine.

Reply:
I'm sorry but I'm unable to follow your reasoning in changing this specific example.

The example is correct here, I just tested it out on `v0.1.7`. To give some context on this specific example, it's to allow users to include notebook files for formatting but exclude them from linting. We don't support versioned documentation so the examples should always reflect as per the latest version.

The reason it fails on `v0.1.0` or earlier is because the `exclude` key didn't exist in those versions.

Reply:
Pretty sure I already explained it, so I'm not sure why you're circling back around to the beginning.